src/ls.ts

```diff
@@ -17,12 +17,12 @@ const supportsLS = (): boolean => {
     if (!localStorage) {
       hasLS = false;
     }
+    hasLS = true;
```

Comment:
Thank you for pointing out the bug. I think this line should come before line 17. Because by placing it here, it suffers from the same problem as before. (i.e. It will always be set to `true`)

Reply:
I have done the needful. Thanks again!

Reply:
Apologies for the slow response. Indeed that was a brain fart of mine! Thanks for fixing 😁